Documentation/guides/k8s-aws-ansible/install.rst

```diff
@@ -0,0 +1,1430 @@
+    
+    aws ec2 create-tags --resources $S3 --tags Key=Name,Value=ciliumSlave3
+    aws ec2 create-tags --resources $S3 --tags Key=role,Value=slave
+    aws ec2 modify-instance-attribute --instance-id $S3 --source-dest-check "{\"Value\": false}"
+
+::
+
+    $ aws ec2 describe-instances --query 'Reservations[*].Instances[*].[InstanceId,PrivateIpAddress,PublicIpAddress,Tags[    0].Value,ImageId,State.Name,Placement.AvailabilityZone,LaunchTime]' --filters Name=tag:Name,Values=cilium* --output text |sort -k     4|grep -v None
+    
+    i-xxxxxxxxxxxxxxxxxxx   192.168.32.11   x.x.x.x ciliumMaster1   ami-xxxxxxxx    running us-west-1a  2017-06-15T13:02:34.000Z
+    i-xxxxxxxxxxxxxxxxxxx   192.168.32.12   x.x.x.x ciliumMaster2   ami-xxxxxxxx    running us-west-1a  2017-06-15T13:02:34.000Z
+    i-xxxxxxxxxxxxxxxxxxx   192.168.32.13   x.x.x.x ciliumMaster3   ami-xxxxxxxx    running us-west-1a  2017-06-15T13:02:34.000Z
+    i-xxxxxxxxxxxxxxxxxxx   192.168.32.21   x.x.x.x ciliumSlave1    ami-xxxxxxxx    running us-west-1a  2017-06-15T13:02:34.000Z
+    i-xxxxxxxxxxxxxxxxxxx   192.168.32.22   x.x.x.x ciliumSlave2    ami-xxxxxxxx    running us-west-1a  2017-06-15T13:02:34.000Z
+    i-xxxxxxxxxxxxxxxxxxx   192.168.32.23   x.x.x.x ciliumSlave3    ami-xxxxxxxx    running us-west-1a  2017-06-15T13:02:34.000Z
+
+
+
+2. Cluster TLS using OpenSSL
+----------------------------
+
+2.1 Kubernetes API Server Keypair
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+::
+
+    core@host:~/cilium$ cd installation/ca/
+    core@host:~/cilium/installation/caNew$ vi openssl.cnf
+    [req]
+    req_extensions = v3_req
+    distinguished_name = req_distinguished_name
+    [req_distinguished_name]
+    [ v3_req ]
+    basicConstraints = CA:FALSE
+    keyUsage = nonRepudiation, digitalSignature, keyEncipherment
+    subjectAltName = @alt_names
+    [alt_names]
+    DNS.1 = kubernetes
+    DNS.2 = kubernetes.default
+    DNS.3 = kubernetes.default.svc
+    DNS.4 = kubernetes.default.svc.cluster.local
+    IP.1 = 192.168.192.1
+    IP.2 = x.x.x.x
+    IP.3 = 192.168.192.10
+    IP.4 = 192.168.32.11
+    IP.5 = 192.168.32.12
+    IP.6 = 192.168.32.13
+    DNS.5 = api.x.x.x.x
+    core@host:~/cilium/installation/caNew$ openssl genrsa -out apiserver-key.pem 2048
+    Generating RSA private key, 2048 bit long modulus
+    ...........................+++
+    ................................................+++
+    e is 65537 (0x10001)
+    core@host:~/cilium/installation/caNew$ openssl req -new -key apiserver-key.pem -out apiserver.csr -subj "/CN=kube-apiserver" -config openssl.cnf
+    core@host:~/cilium/installation/caNew$ openssl x509 -req -in apiserver.csr -CA ca.pem -CAkey ca-key.pem -CAcreateserial -out apiserver.pem -days 365 -extensions v3_req -extfile openssl.cnf
+    Signature ok
+    subject=/CN=kube-apiserver
+    Getting CA Private Key
+
+2.2 Kubernetes Worker Keypairs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Needs a description that we are generating the key for the slaves here and what the key will be used for